tools/vfsstat.py

```diff
@@ -51,34 +59,48 @@ def usage():
 
 BPF_ARRAY(stats, u64, S_MAXSTAT);
 
-static void stats_increment(int key) {
+static void stats_try_increment(int key) {
+    u32 pid = bpf_get_current_pid_tgid() >> 32;
```

Comment:
Move this line into `PID_FILTER`, so that we save a helper call if no PID filter applied.

Reply:
done，thanks.